packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -64,6 +66,26 @@ enum _TextSelectionHandlePosition { start, end }
 /// Used by [TextSelectionOverlay.onSelectionOverlayChanged].
 typedef TextSelectionOverlayChanged = void Function(TextEditingValue value, Rect caretRect);
 
+/// Signature for when a pointer has started dragging to select text.
+///
+/// The [details] argument is the same as the one passed to [DragGestureRecognizer.onStart].
+typedef DragSelectionStartCallback = void Function(DragStartDetails details);
+
+/// Signature for when a pointer that's dragging to select text has moved again.
+///
+/// The first argument [startDetails] contains the details of the event that
+/// initiated the dragging.
+///
+/// The second argument [updateDetails] contains the details of the current
+/// pointer movement. It's the same as the one passed to [DragGestureRecognizer.onUpdate].
```

Comment:
ummm this one is unfortunate. I don't have a better solution to suggest. I would document why this one exists (i.e. DragUpdateDetails only give momentary deltas).

Reply:
This way it's easier for text fields since they don't have to store the start position.